content/docs/0.13.x/operate/user_management/oauth_secrets/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: OpenID Session Secrets
+description: Manage session and database encryption secret of your Keptn Bridge.
+weight: 30
+---
+
+In this section, the management of the session and database encryption secrets of the Keptn Bridge is explained.
+
+- `session secret`: This secret is used to hash the session of the user.
+- `database encryption secret`: This secret is used to encrypt the session in the database.
```

Comment:
```suggestion
- `session secret`: Used to hash the session of the user.
- `database encryption secret`: Used to encrypt the session in the database.
```
"This secret is" in these two items seems like superfluous noise but you may disagree.